doc/man3/OSSL_CMP_ITAV_set0.pod

```diff
@@ -29,21 +29,21 @@ ITAV is short for InfoTypeAndValue. This type is defined in RFC 4210
 section 5.3.19 and Appendix F. It is used at various places in CMP messages,
 e.g., in the generalInfo PKIHeader field, to hold a key-value pair.
 
-OSSL_CMP_ITAV_create() creates a new OSSL_CMP_ITAV structure and fills it in.
-It combines B<OSSL_CMP_ITAV_new()> and B<OSSL_CMP_ITAV_set0>.
+OSSL_CMP_ITAV_create() creates a new B<OSSL_CMP_ITAV> structure and fills it in.
```

Comment:
Here the OSSL_CMP_ITAV is in bold but elsewhere in the document there are other typedefs like OSSL_CMP_CTX and ASN1_TYPE kept in regular font.

Reply:
Thanks, I'll fix them

Reply:
Done